squire/settings.py

```diff
@@ -224,13 +232,13 @@
 COMMITTEE_FULL_NAME = 'Hackmanite Turbo Typing Programming Squad'
 
 # People who get error code notifications if Debug = False
-ADMINS = [(APPLICATION_NAME + ' Admin', 'https@kotkt.nl')] # NB: This email should be changed to something else
+ADMINS = [(APPLICATION_NAME + ' Admin', 'https@kotkt.nl')] # TODO: This email should be changed to something else
 
 # The email address that error messages come from, such as those sent to ADMINS and MANAGERS.
-SERVER_EMAIL = '{0} Error <error@{1}.kotkt.nl>'.format(APPLICATION_NAME, APPLICATION_NAME.lower())
+SERVER_EMAIL = f'{APPLICATION_NAME} Error <error@{APPLICATION_NAME.lower()}.kotkt.nl>'
 
 # Default email address to use for various automated correspondence from the site manager(s).
-DEFAULT_FROM_EMAIL = '{0} <noreply@{1}.kotkt.nl>'.format(APPLICATION_NAME, APPLICATION_NAME.lower())
+DEFAULT_FROM_EMAIL = f'{APPLICATION_NAME} <noreply@{APPLICATION_NAME.lower()}.kotkt.nl>'
```

Comment:
Oof, that email will not even resolve. Why not `noreply-{APPLICATION_NAME.lower()}@kotkt.nl`?